remappings.txt

```diff
@@ -1,6 +1,3 @@
 forge-std/=lib/forge-std/src/
 openzeppelin-contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/
 openzeppelin-contracts/=lib/openzeppelin-contracts/
-lib/openzeppelin-contracts:@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
```

Comment:
I removed these remappings because they were causing build errors for me and I figured that would be the case for others as well. I can revert and investigate my local setup if it causes issues for others